package com.challenge.exceptions;

public class ExcepcionGeneral extends AssertionError {

    public ExcepcionGeneral(String message, Throwable cause) {
        super(message, cause);
    }

}
